### ISSUE TITLE
Programmatic way to launch Storybook

### ISSUE THREAD

**Issue by mquandalle** [storybooks/react-storybook#146]
In an application I’m developing I have a root directory called `tools/` that contains various things to spawn Webpack dev server, or Graph_i_QL, or run scripts. I would like to define Storybook configuration in `tools/storybook.js` and be able to call it using JavaScript methods instead of using the `start-storybook` command line interface. That would allow things like:

```
$ npm run dev
Application is available at http://localhost:3000
GraphiQL is available at http://localhost:4000
StoryBook is available at http://localhost:5000
```

How to do that?

**Comment by arunoda** [comment]
You can change the config directory using this API: https://github.com/kadirahq/react-storybook/blob/master/docs/configure_storybook.md#configuration-directory

You may use shelljs to run the storybook command.

**Comment by mquandalle** [comment]
But can I change the `config.js` filename?

**Comment by arunoda** [comment]
We have many config files, so it's always better to store them in a directory. (even if you've just the `config.js` file)

So, I think changing that's not ideal.
But I'm open to find a way to run the storybook manually.

**Comment by mnmtanish** [comment]
Hi @mquandalle 
A storybook express middleware is available from `v1.28.0`.

```js
import storybook from '@kadira/storybook/dist/server/middleware';

// ...
app.use(storybook(configDirectory));
// ...
```

Check this [line](https://github.com/kadirahq/react-storybook/blob/3a21d57d30f7a7f0252205f9dfc6dc820fb46e94/src/server/index.js#L49)

Related: https://github.com/kadirahq/react-storybook/issues/208
